### PR TITLE
SOC-1795 | Add GA pageviews tracking for discussions

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-1795

Ping @rogatty @rafalkalinski

### CODE REVIEW

File: front/scripts/main/app.js [hunk 1]
```diff
@@ -170,6 +170,9 @@ App.initializer({
 				window.location.pathname === '/'
 			) {
 				return 'home';
+			} else if (window.location.pathname.startsWith('/d/')) {
```
Comment (rogatty) [inline front/scripts/main/app.js]: Won't work in IE11, see http://kangax.github.io/compat-table/es6/#test-String.prototype_methods_String.prototype.startsWith

Comment (rafalkalinski) [inline front/scripts/main/app.js]: indexOf?

Comment (kvas-damian) [inline front/scripts/main/app.js]: thanks for catching that

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**